### PR TITLE
doc: edit README, organize chat resources

### PR DESCRIPTION
**Pull request by Trott**
This is inspired by #126.

### CODE REVIEW

**Comment by lordKnighton** [comment]
#LGTM

File: README.md [hunk 1]
```diff
@@ -1,15 +1,20 @@
...
 
-## Please feel free to ask
+## Chat Resources
 
-[Please click here](https://github.com/nodejs/help/issues/new) to create a new issue if you have any questions. :smile:
+If you would prefer to get help via live chat rather than the issue tracker in
+this repository, you can try:
+
+* [Freenode #node.js channel](https://webchat.freenode.net/?channels=node.js&uio=d4)
+* [Gitter Node.js room](https://gitter.im/nodejs/help)
```
Comment (TheAlphaNerd) [inline README.md]: I think we should wait to see how #126 works out before linking gitter

Comment (Trott) [inline README.md]: Argh! I meant to link to the same Gitter room as is [in the nodejs/node repo README](https://github.com/nodejs/node#resources-for-newcomers) but apparently, I messed up. Let me update....

Comment (Trott) [inline README.md]: OK, fixed.

**Comment by JungMinu** [comment]
LGTM

**Pull request merged**